src/lib.rs

```diff
@@ -476,6 +499,35 @@ pub mod pallet {
 			let _ = frame_system::Pallet::<T>::dec_providers(&supersig_id);
```

Comment:
Just above:
```
let _ = T::Currency::transfer(
				&supersig_id,
				&beneficiary,
				balance,
				ExistenceRequirement::AllowDeath,
			);
```

Just use the `?` please.
Yeah, maybe right now you are sure it cannot fail, but the code will move in the future. Doesn't cost nothing to return the error if there is an error, rather than just ignoring it.
Do it for the whole pallet. Handle your Return. Always. You really must have a really good reason for not doing it

Reply:
Okay.
Do i keep the comments saying it will not fail ?
I also added `#[transactional]`